x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/SecurityIndexManager.java

```diff
@@ -97,10 +93,6 @@ public SecurityIndexManager freeze() {
         return new SecurityIndexManager(null, indexName, indexState);
     }
 
-    public static List<String> indexNames() {
-        return Collections.unmodifiableList(Arrays.asList(SECURITY_INDEX_NAME, INTERNAL_SECURITY_INDEX));
-    }
-
```

Comment:
These lines are responsible for most of the changes. Security index names had to be moved from `x-pack:plugin:security` to `x-pack:plugin:core` . I have created `org.elasticsearch.xpack.core.security.index.SystemIndicesNames`.